crates/bytes-util/src/cow/bytes/mod.rs

```diff
@@ -19,6 +20,17 @@ pub enum BytesCow<'a> {
     Bytes(Bytes),
 }
 
+impl Debug for BytesCow<'_> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Self::Slice(slice) => Debug::fmt(slice, f),
+            Self::StaticSlice(slice) => Debug::fmt(slice, f),
+            Self::Vec(bytes) => Debug::fmt(bytes, f),
+            Self::Bytes(bytes) => Debug::fmt(bytes, f),
+        }
+    }
+}
```

Comment:
I think it would be better to instead debug this as a `&[u8]` or to implement it as a raw byte literal debug. `b"\x0d whatever"`